pettingzoo/utils/random_demo.py

```diff
@@ -3,26 +3,20 @@
 import numpy as np
 
 
-def random_demo(env, render=True, cycles=100000000):
+def random_demo(env, render=True, episodes=1):
     '''
     Runs an env object with random actions.
     '''
 
-    env.reset()
-
-    if hasattr(env, 'display_wait'):
-        display_wait = env.display_wait
-    else:
-        display_wait = 0.0
-
     total_reward = 0
     done = False
+    run_episodes = 0
 
-    for cycle in range(cycles):
+    while True:
```

Comment:
Better code style would be `while run_episodes < episodes:` Then after the loop, have your `if render: env.cose()` check